docs/auth-sso/okta-integration-guide.mdx

```diff
@@ -0,0 +1,162 @@
+
+### 🟠 Okta Account Access
+
+Ensure you have either **Super Admin** or **Organization Admin** privileges.
+
+<details>
+    <summary><p>Why is this required?</p></summary>
+    <p>Having <strong>Super Admin</strong> or <strong>Organization Admin</strong> privileges enables you to:</p>
+    <ul>
+        <li>Add or edit attributes and their mappings via <code>Directory &gt; Profile Editor</code>.</li>
+        <li>Add or edit Identity Providers (IdPs) through <code>Security &gt; Identity Providers</code>.</li>
+        <li>Add or edit routing rules in <code>Security &gt; Identity Providers &gt; Routing Rules</code>.</li>
+    </ul>
+</details>
+
+### 🟠 Feature Verification
+
+Confirm that the following features are active on your account:
+
+<details>
+    <summary><p>OpenID Connect IdP</p></summary>
+    <p>Verify this by checking if you can navigate to <code>Security &gt; Identity provider &gt; Add Identity Provider &gt; Add OpenID Connect IdP</code>. If this option is unavailable, you will need to contact Okta support to enable it.</p>
+</details>
+
+<details>
+    <summary><p>Routing Rules</p></summary>
+    <p>Ensure the "Routing Rules" tab is visible under <code>Security &gt; Identity Providers</code>. If it is missing, contact Okta support to request its activation on the Identity Provider page.</p>
+</details>
+
+---
+
+## Prepare Okta for Integration
+
+### 🟠 Update User Profile Attributes
+
+To enable Okta to automatically direct users registered with Beyond Identity towards a passwordless authentication flow, define a custom attribute named "byndidRegistered" on an Okta User's profile.
+
+<Arcade clip={Clip.OktaConfigureUserSettings} />
+
+### 🟠 Create a Beyond Identity Group
+- Create a group and add users to it
+
+TODO: Add arcade here
+
+### 🟠 Generate API Token
+- Security -> API -> Tokens -> Create token
+- Keep this somewhere safe for now
+
+TODO: Add arcade here
+
+---
+
+## Set Up Beyond Identity
+### 🔵 Create Beyond Identity Account
+- Go to marketing website
+- Create BI account
+
+TODO: Add arcade here
+
+### 🔵 Integrate with Okta Directory
+- Go to directory integrations -> Add Directory -> Okta
+- Enter your API token where requested
+- Select bearer token for SCIM -> click save
+
+TODO: Add arcade here
+
+### 🔵 Create IdP application
+- Auth & SSO -> Add Identity Provider
+- Keep Values Handy
+
+TODO: Add arcade here
+
+---
+
+## Synchronize Users via SCIM
+
+### 🟠 Enable syncing of users from Okta into Beyond Identity:
+- In Okta Set up a SCIM application
+- Paste in SCIM information from BI directory integration
+
+TODO: Add arcade here
+
+---
+
+## Configure Identity Provider
+
+### 🟠 Set Up OIDC Identity Provider
+- Set up OIDC identity provider.
+- Copy values from BI.
+- Save Identity provider
+- Set up routing rules for byndidRegistered attribute
+
+TODO: Add arcade here
+
+---
+
```

Comment:
Should there be a section to test this with a small set of users?

Reply:
Yes good idea. I've added a note to include that. Thank you.